unit_test/plugin_examples/good_plugin/help.md

```diff
@@ -46,7 +46,7 @@ Example input:
 
 |Name|Type|Required|Description|
```

Comment:
Looks like you're adding a column with examples but the title row didn't have a Example title added to it?